tests/event/test_event.py

```diff
@@ -1,26 +1,55 @@
 import pytest
 from jinja2 import Template
 
+from mobilizon_reshare.config.config import get_settings
+
+
+@pytest.fixture()
+def set_locale(locale):
+    settings = get_settings()
+    old_locale = settings["locale"]
+    yield get_settings().update({"locale": locale})
+    settings.update({"locale": old_locale})
+
 
 @pytest.fixture()
 def simple_template():
     return Template(
         (
             "{{name}}|{{description}}|{{location}}|{{begin_datetime.strftime('%Y-%m-%d, %H:%M')}}"
-            "|{{last_update_time.strftime('%Y-%m-%d, %H:%M')}}"
+            "|{{end_datetime.strftime('%Y-%m-%d, %H:%M')}}"
+        )
+    )
+
+
+@pytest.fixture()
+def template_with_locale():
+    return Template(
+        (
+            "{{name}}|{{description}}|{{location}}|{{begin_datetime.format('DD MMMM, HH:mm', locale=locale)}}"
+            "|{{end_datetime.format('DD MMMM, HH:mm', locale=locale)}}"
         )
     )
 
 
 def test_fill_template(event, simple_template):
     assert (
         event._fill_template(simple_template)
-        == "test event|description of the event|location|2021-01-01, 11:30|2021-01-01, 11:30"
+        == "test event|description of the event|location|2021-01-01, 11:30|2021-01-01, 12:30"
```

Comment:
Why is this happening? Not a problem, it's just that timezones are already becoming my nightmare

Reply:
it's not a timezone issue: in the template I was using the last_accessed_time instead of the end_datetime